front/ui/storybook/stories/ui-charts/trackOccupancyDiagram/rendering.stories.tsx

```diff
@@ -1,286 +1,38 @@
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { TrackOccupancyStandalone } from '@osrd-project/ui-charts';
 
 import type { Meta, StoryObj } from '@storybook/react';
 
-import occupancyZones from './assets/occupancyZones';
-import tracks from './assets/tracks';
-import { TimeCaptions } from '../../../../ui-charts/src/spaceTimeChart/components/TimeCaptions';
-import { useCanvas, useDraw } from '../../../../ui-charts/src/spaceTimeChart/hooks/useCanvas';
-import { useMouseInteractions } from '../../../../ui-charts/src/spaceTimeChart/hooks/useMouseInteractions';
-import { useMouseTracking } from '../../../../ui-charts/src/spaceTimeChart/hooks/useMouseTracking';
-import { useSize } from '../../../../ui-charts/src/spaceTimeChart/hooks/useSize';
-import { DEFAULT_THEME } from '../../../../ui-charts/src/spaceTimeChart/lib/consts';
-import {
-  CanvasContext,
-  MouseContext,
-  SpaceTimeChartContext,
-} from '../../../../ui-charts/src/spaceTimeChart/lib/context';
-import type {
-  MouseContextType,
-  SpaceTimeChartContextType,
-  PickingElement,
-  SpaceTimeChartTheme,
-} from '../../../../ui-charts/src/spaceTimeChart/lib/types';
-import {
-  getTimeToPixel,
-  getSpaceToPixel,
-  getDataToPoint,
-  getPixelToTime,
-  getPixelToSpace,
-  getPointToData,
-  spaceScalesToBinaryTree,
-} from '../../../../ui-charts/src/spaceTimeChart/utils/scales';
-import {
-  TrackOccupancyManchette,
-  TrackOccupancyCanvas,
-} from '../../../../ui-charts/src/trackOccupancyDiagram/index';
-import { KebabHorizontal } from '../../../../ui-icons/src/index';
-import { OPERATIONAL_POINTS } from '../spaceTimeChart/helpers/paths';
+import OCCUPANCY_ZONES from './assets/occupancyZones';
+import { TRACKS } from '../manchetteWithSpaceTimeChart/assets/trackOccupancyData';
```

Comment:
This change breaks the build at this commit
```suggestion
import TRACKS from './assets/tracks';
```